src/nrniv/bbsavestate.cpp

```diff
@@ -803,12 +799,6 @@ extern "C" void bbss_restore_done(void* bbss) {
 	npe->savestate_restore(t, net_cvode_instance);
 	delete npe;
 	nrn_spike_exchange(nrn_threads);
-#if BGPDMA
-	// only necessary if multisend method is using two subintervals
```

Comment:
This is a sign to me that the removal removed at least some aspects of the BlueGene/P independent aspects of multisend. The important consideration is
```
nrniv/netpar.cpp:// BGPDMA can be 0,1,2,3,6,7
nrniv/netpar.cpp:// (BGPDMA & 1) > 0 means multisend ISend allowed
nrniv/netpar.cpp:// (BGPDMA & 2) > 0 means multisend Blue Gene/P DMA allowed
nrniv/netpar.cpp:// (BGPDMA & 4) > 0 means multisend Blue Gene/P DMA Record Replay allowed
```

Reply:
we want to allow ISend implementation of the multisend method and also make it enableable from cmake. I.e all code that compiles for  BGPDMA values of 0 and 1 needs to be retained. All the rest can be removed. So we want an ISend/IRecv
implemented multisend which works for 1 and 2 phase (TWOPHASE) and 1 and 2 interval (BGP_INTERVAL)

Reply:
hey @nrnhines 

I deleted TWOPHASE so I should keep BGPDMA with values 0 and 1 and BGP_INTERVAL with value 1 and 2,

Right?

Reply:
IIUC, we have to retain: BGPDMA values of 0 and 1`, 1 and 2 phase (TWOPHASE) and 1, 2 interval (BGP_INTERVAL)

Reply:
TWOPHASE has been deleted in a previous PR: https://github.com/neuronsimulator/nrn/pull/1344